### PR TITLE
Feature/nvanny/1209376169 delete modal

### PR DESCRIPTION
**Pull request by nvanny**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
Created the delete modal

### Testing
How did you confirm your changes worked? 
Looked at the delete modal functionality when clicking the trash can

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| [TODO Screenshot Title] |
| --- |
| [TODO Screenshot Link] |
image

### CODE REVIEW

File: app/views/userInterface/index.ejs [hunk 1]
```diff
@@ -13,8 +13,33 @@
         <script src="javascripts/materializejs/materialize.min.js"></script>
         <link href="stylesheet/datatable/datatable.min.css" rel="stylesheet">
         <link href="stylesheet/dashboard.css" rel="stylesheet" type="text/css">
+        <link href="stylesheet/deleteModal.css" rel="stylesheet">
         <link href="stylesheet/clientsModal.css" rel="stylesheet">
         <link href="stylesheet/notifModal.css" rel="stylesheet">
+
+        <style>
```
Comment (mingkaili) [inline app/views/userInterface/index.ejs]: Maybe put the style into its own css file?

**Review by mingkaili — CHANGES_REQUESTED** [review]

**Review by eliasfang — APPROVED** [review]
This looks great!

File: app/views/userInterface/index.ejs (hunk 1 above)
Comment (nvanny) [reply app/views/userInterface/index.ejs]: just moved it, let me know if that works

**Review by mingkaili — APPROVED** [review]
Great Great work! You can start on the backend functionalities now

**Pull request merged**